src/linux/system.rs

```diff
@@ -750,6 +769,10 @@ fn get_system_info_linux(info: InfoType, path: &Path, fallback_path: &Path) -> O
     let info_str = match info {
         InfoType::OsVersion => "DISTRIB_RELEASE=",
         InfoType::Name => "DISTRIB_ID=",
+        InfoType::DistributionID => {
```

Comment:
Why adding this variant if it's never used?

Reply:
Ah nevermind, didn't see it was created into the `distribution_id` method. My bad.

Reply:
It's used on the line 753:

```
          let info_str = match info { ‣&str
              InfoType::Name => "NAME=",
              InfoType::OsVersion => "VERSION_ID=",
              InfoType::DistributionID => "ID=",
          };
```